File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/Corruption.java
package fr.jbme.raiderioapp.data.model.character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Corruption {

    @SerializedName("added")
    @Expose
    public Integer added;
    @SerializedName("resisted")
    @Expose
    public Integer resisted;
    @SerializedName("total")
    @Expose
    public Integer total;
    @SerializedName("cloakRank")
    @Expose
    public Integer cloakRank;
    @SerializedName("spells")
    @Expose
    public List<Spell> spells = null;

}

File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/Corruption.kt
package fr.jbme.raiderioapp.data.model.character

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Corruption(
    @SerializedName("added")
    @Expose
    var added: Int? = null,

    @SerializedName("resisted")
    @Expose
    var resisted: Int? = null,

    @SerializedName("total")
    @Expose
    var total: Int? = null,

    @SerializedName("cloakRank")
    @Expose
    var cloakRank: Int? = null,

    @SerializedName("spells")
    @Expose
    var spells: List<Spell>? = null
)